includes/polling/applications/memcached.inc.php

```diff
@@ -1,6 +1,53 @@
 <?php
 
-$data = $agent_data['app']['memcached'][$app['app_instance']];
+if (!empty($agent_data['app']['memcached'])) {
+    $data = $agent_data['app']['memcached'][$app['app_instance']];
+} else {
+    $options = '-O qv';
+    $oid     = 'nsExtendOutputFull.9.109.101.109.99.97.99.104.101.100';
```

Comment:
Use the full number `.1.3.6.1.4.1.8072.1.3.2.3.1.2.9.109.101.109.99.97.99.104.101.100` or specify the `NET-SNMP-EXTEND-MIB` mib.